app/src/main/java/io/homeassistant/companion/android/sensors/SensorReceiver.kt

```diff
@@ -115,6 +115,7 @@ class SensorReceiver : SensorReceiverBase() {
         AudioManager.ACTION_MICROPHONE_MUTE_CHANGED to AudioSensorManager.micMuted.id,
         AudioManager.ACTION_SPEAKERPHONE_STATE_CHANGED to AudioSensorManager.speakerphoneState.id,
         AudioManager.RINGER_MODE_CHANGED_ACTION to AudioSensorManager.audioSensor.id,
+        AudioSensorManager.VOLUME_CHANGED_ACTION to AudioSensorManager.volMusic.id,
```

Comment:
I am not sure if this is correct. According to the docs this intent gets fired for each and every volume stream and not just the media stream. The way this is now if the user does not have the volume music sensor but has any of the other volume sensors enabled then they will not update immediately.

https://android.googlesource.com/platform/frameworks/base/+/8bd69610aafc6995126965d1d23b771fe02a9084/media/java/android/media/AudioManager.java#102

Reply:
I've just tested it and you're right, other volume sensors won't update immediately if the music volume sensor is disabled.

Reply:
Maybe you have a solution in mind that you'd prefer?

Reply:
currently its a 1 to 1 mapping here, not all intents are added here like network changes which is more similar to the intent you have added as multiple sensors can be updated by it.

https://github.com/home-assistant/android/blob/f73a9dddec423a3ef66f02f70175168733c501d6/app/src/main/java/io/homeassistant/companion/android/HomeAssistantApplication.kt#L129

the best place to add something else unique would be in `SensorReceiverBase` where we do additional processing like the skippable actions.

Reply:
I came up with a bit different solution. Please check it in (26d496aa802af3bc81619df6ec94aaf24716379a) and let me know if you like it or not. If not, I'm open to dropping this and implementing something based on your previous hints.

Reply:
great solution :)